src/Sarif.WorkItems/SarifWorkItemModel.cs

```diff
@@ -56,23 +56,11 @@ public SarifWorkItemModel(SarifLog sarifLog, SarifWorkItemContext context = null
             //
             this.CommentOrDiscussion = $"Default {nameof(this.CommentOrDiscussion)}";
 
-
-            // TODO: Provide a useful SARIF-derived issue body. Note that there
-            //       is a client-specific consideration here in that there is much
-            //       less pressure to provide an ADO rendering of the scan results 
-            //       in the bug body, as we can integrate the SARIF results viewing
-            //       web control in that environment. So for this scenario, a very
-            //       simple boilerplate text could be sufficient. For GH, we would
-            //       prefer the issue body to provide more details. Another clear
-            //       distinction centers on differences in rendering format
-            //       (markdown vs. HTML). We only have a single tracking issue below - 
-            //       in the event, this work is likely to be broken aparts into 
-            //       multiple tasks, each of which may entail breaking changes to 
-            //       models/API.
-            //
-            //       https://github.com/microsoft/sarif-sdk/issues/1757
-            //
-            this.BodyOrDescription = $"Default {nameof(this.BodyOrDescription)}";
+            this.BodyOrDescription = sarifLog.CreateWorkItemDescription();
+            if (string.IsNullOrEmpty(this.BodyOrDescription))
```

Comment:
>IsNullOrEmpty [](start = 23, length = 13)

I think we can live with an empty body or description if your helper bellies up. So suggest deleting this setting of default entirely in that condition.